### PR TITLE
Re-licensed under MPL 2.0

### PR DESCRIPTION
**Pull request by murat-dogan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
       "url": "https://github.com/paullouisageneau"
     }
   ],
-  "license": "LGPL",
+  "license": "MPL 2.0",
```
Comment (paullouisageneau) [inline package.json:44]: Possibly just a nitpick, but I think this should be the SPDX license identifier, so `"MPL-2.0"`.

I don't know if it's important for npm.

**Comment by paullouisageneau** [comment]
Great :tada:

**Comment by paullouisageneau** [comment]
Great, thanks :tada:

**Comment by murat-dogan** [comment]
Thanks for the hint

**Pull request merged**